### ISSUE TITLE
Don't override head styles

### ISSUE THREAD

**Issue by oscarmorrison** [oscarmorrison/md-page#19]
If styles exist in the head already, do not override this styles.

Make the md-page styles the least important of all styles

**Comment by p1ho** [comment]
I think... in HTML, if a style is repeatedly defined, the last one should take precedence.
For example,
```HTML
<style>
  h1 {color: red;}
</style>
<style>
  h1 {color: blue;}
</style>
<h1>hi</h1>
```
The h1 text will be blue.

So if you want the styles you defined in [`script.js`](https://github.com/p1ho/md-page/blob/master/script.js) to take the least precedence, I would make that the first child in your head.
You could do this by replacing [`document.head.appendChild(sheet)`](https://github.com/p1ho/md-page/blob/master/script.js#L13) with `document.head.prepend(sheet)`

**Comment by oscarmorrison** [comment]
@p1ho sounds good. https://github.com/oscarmorrison/md-page/blob/master/script.js#L13 prepend here instead. Want to open a PR and give this a try?

**Comment by p1ho** [comment]
I just checked, according to my local testing, external stylesheets (with `<link>`) and inline style tags (`<style>`) have the same level of priority, so yes, I think changing that one line will do the trick.

I've opened a PR: #37

**Comment by oscarmorrison** [comment]
Fixed y #37
